lnwallet/channel.go

```diff
@@ -153,22 +153,6 @@ func (e *ErrCommitSyncLocalDataLoss) Error() string {
 		e.CommitPoint.SerializeCompressed())
 }
 
-// channelState is an enum like type which represents the current state of a
-// particular channel.
-// TODO(roasbeef): actually update state
-type channelState uint8
```

Comment:
ok nvm so the whole thing is removed - in that case can we do all these removals in one commit? Easier to follow and reason about, and put that `channelDispute -> channelClosed` in its own commit to ephemsize the bug fix.

Reply:
Yeah I don't mind that. The choice for micro commits was to consider each state removal in isolation, but perhaps that was the wrong approach.

Reply:
yeah I like that we are creating small commits so it's reviewer-friendly, but I guess it's not always the smaller the better. Think it should be small but also contain the necessary changes to avoid confusion. Anyway non-blocking this time, think we'll get better at this in the future.